### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by por4por** [julik/sylens#24]
I see a crop happening on positive distortion. (.0107) It crops all around the edges.  Have you seen this issue?

**Comment by julik** [comment]
Yes, it also happens with zooming. I will look into it. I think some LUT recomputation in the SyDistorter might be the cause.

**Comment by por4por** [comment]
Any chance this has been fixed? Thanks!

**Comment by julik** [comment]
Nope, haven't had time for any SyLens work recently and haven't done any Nuke for a few years now.
